src/App.tsx

```diff
@@ -1,11 +1,29 @@
-import { useState } from 'react';
+import { useState, useCallback } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [query, setQuery] = useState('');
```

Comment:
Are you sure you need this state in the App component? As I see you use it only inside FindMovie, so I suggest to move it there